string country_name
---
string capital_name
bool success
